webclient/components/pt-info/single/1time-1row-mField/phq9/db/client-side/static-data/search-phrases-ct.vue

```diff
@@ -1,11 +1,14 @@
+<!-- To understand how this works read components/core/search-phrases/load-search-phrases-and-handle-selection.vue -->
 <template>
-  <div></div>
+  <div><tblMasterListOfPhq9Ct /></div>
```

Comment:
Why is this needed?

Reply:
in vue to add child comonent, needs to bind the component name in the template file as a with < child comonent />